packages/amplify-e2e-tests/package.json

```diff
@@ -44,7 +44,8 @@
     "yargs": "^15.1.0"
   },
   "devDependencies": {
-    "ts-node": "^8.9.0"
+    "ts-node": "^8.9.0",
+    "jest": "^27.2.4"
```

Comment:
The CLI depends on jest "^26.6.3" in the root package.json. We should keep these consistent.

Reply:
While we're at it, we have 3 different ts-node versions defined in package.json. This could at least be updated to use `8.10.2`

Reply:
Done.